newIDE/app/src/SceneEditor/Toolbar.js

```diff
@@ -38,9 +38,9 @@ export class Toolbar extends PureComponent<Props> {
     return (
       <>
         <ToolbarCommands
-          openObjectsList={this.props.openObjectsList}
-          openObjectGroupsList={this.props.openObjectGroupsList}
-          openPropertiesPanel={this.props.openProperties}
+          toggleObjectsList={this.props.toggleObjectsList}
+          toggleObjectGroupsList={this.props.toggleObjectGroupsList}
+          togglePropertiesPanel={this.props.toggleProperties}
```

Comment:
ToolbarCommands component is used for bind ab action to shortcuts.
In your case this code isn't complete, because props in ToolbarCommands are not complete.

See if you press O for open the object editor the app should crash.

Reply:
now toggle functionality linked to shortcut keys.